BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -289,6 +290,24 @@ - (void)start
     [[BOXAPIOperation APIOperationGlobalLock] unlock];
 }
 
+- (BOOL)shouldUseSessionTask
+{
+    return YES;
+}
+
+- (NSURLSessionTask *)sessionTask
+{
+    if (_sessionTask == nil) {
```

Comment:
updated to method createSessionTask instead, so it's called on-demand when we need a valid sessionTask